content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -493,4 +492,4 @@ the `.spec.replicas` field automatically.
   Read the {{< api-reference page="workload-resources/stateful-set-v1" >}}
   object definition to understand the API for stateful sets.
 * Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
-  you can use it to manage application availability during disruptions.
\ No newline at end of file
+  you can use it to manage application availability during disruptions.
```

Comment:
@ayushpatil2122, does this new addition provide any changes to the original content.

Reply:
no